client/app/bundles/course/assessment/submission/reducers/liveFeedback.js

```diff
@@ -1,125 +1,172 @@
 import { produce } from 'immer';
 
+import { suggestionsTranslations } from 'course/assessment/submission/translations';
+
 import actions from '../constants';
 
-const initialState = {
+export const LIVE_FEEDBACK_LOCAL_STORAGE_KEY = 'liveFeedbackStateLocalStorage';
+
+const loadStateFromLocalStorage = () => {
+  try {
+    const serializedState = localStorage.getItem(
+      LIVE_FEEDBACK_LOCAL_STORAGE_KEY,
+    );
+    return serializedState ? JSON.parse(serializedState) : undefined;
+  } catch (error) {
+    return undefined;
+  }
+};
+
+const saveStateToLocalStorage = (state) => {
+  const serializedState = JSON.stringify(state);
+  localStorage.setItem(LIVE_FEEDBACK_LOCAL_STORAGE_KEY, serializedState);
+};
+
+const initialState = loadStateFromLocalStorage() || {
   feedbackUrl: null,
   feedbackByQuestion: {},
 };
-export default function (state = initialState, action) {
+
+const getRandomSuggestions = () => {
+  const suggestionValues = Object.values(suggestionsTranslations);
+  const shuffled = suggestionValues.sort(() => 0.5 - Math.random());
+  // TODO: Temporarily change to show no suggestions
+  return shuffled.slice(0, 0);
+};
```

Comment:
Don't implement your own shuffle, your algorithm is not truly random.

You can use our `lodash` library's `shuffle`.